types/light_info.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import "github.com/tendermint/tendermint/libs/bytes"
+
+// Info about the status of the light client
+type LightClientInfo struct {
+	Primary           string         `json:"primary"`
+	Witnesses         []string       `json:"witnesses"`
```

Comment:
Are these addresses? Keys?

Reply:
Right now it is the IP, but from what I understood down the road, with a p2p provider it will be a combination of nodeID and IP address. So perhaps renaming it to ID would make things more clear for the time being.

Reply:
Makes sense. Perhaps `WitnessID` ?